data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/metrics/Metrics.java

```diff
@@ -77,6 +77,12 @@
    */
   public static final String EVENT_PROCESSING_LATENCY = "event_processing_latencies";
 
+    /**
+   * @link https://knative.dev/docs/eventing/observability/metrics/eventing-metrics/
+   * @see Metrics#discardedEventCount(io.micrometer.core.instrument.Tags)
+   */
+  public static final String DISCARDED_EVENTS_COUNT = "discarded_event_count";
```

Comment:
nit: motivation is the same as the other comment.
```suggestion
  public static final String DISCARDED_EVENTS_COUNT = "discarded_invalid_event_count";
```